numba/tests/test_array_methods.py

```diff
@@ -232,6 +232,12 @@ def array_real(a):
 def array_imag(a):
     return np.imag(a)
 
+def np_clip(a, a_min, a_max, out=None):
+    return np.clip(a, a_min, a_max, out)
+
+def array_clip(a, a_min=None, a_max=None, out=None):
+    return a.clip(a_min, a_max, out)
```

Comment:
It'd be good to also include variants where the kwargs are explicitly wired, e.g.
```python
def np_clip_kwargs(a, a_min, a_max, out):
    return np.clip(a, a_min, a_max, out=out)
```
and the same for `array_clip` so as to ensure the kwargs are behaving as expected (along with test modifications to check).

Reply:
Added.

Reply:
thanks